Gemfile

```diff
@@ -115,7 +118,7 @@ unless ENV['APPLIANCE']
   end
 
   group :development, :test do
-    gem "rspec-rails",      "~>3.0"
+    gem "rspec-rails",      "~>3.5.0.pre",                 :git => "git://github.com/rspec/rspec-rails", :branch => "master"
```

Comment:
Wait, my bad. 3.5.0 isn't released yet, but there is a [released patch version (3.4.1) for Rails 5](https://github.com/rspec/rspec-rails/commits/v3.4.1) that we should now use here instead, @matthewd